get-coverage-percent/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "get-coverage",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "ncc build index.js --license licenses.txt",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
can the Github Action Licenses spreadsheet in shared drive be updated to include this one?

Reply:
I went to double check that we were using BSD-3 for our other repos, and noticed that answers-headless also says ISC here... probably due to it being the default for npm init. I made an item to make sure our other repos have licenses in sync with their package.json field https://yexttest.atlassian.net/browse/SLAP-2154
headless react doesn't have a license field

Reply:
I copied what core has for the license field and also the bsd-3 license. I didn't see a need to add the preamble but I can if you feel like it's needed

Reply:
oops I wasn't clear. I meant that we might want to update [this spreadsheet](https://docs.google.com/spreadsheets/d/1LTZgKwtAYTA6M_g5iZrg5BAmF8bW3RPBzq33MZaLQ20/edit#gid=0)

Reply:
Oh I wasn't sure if ISC was allowed?

Reply:
If it is I can update

Reply:
I think ISC is only allowed for some use cases, so if we're licensing under BSD-3, that's probably better and we can just list it on that spreadsheet.

(this spreadsheet is basically trying to do what Snyk does and keep track of the licenses of the dependencies in this repo since there isn't a top-level package.json. so whenever a new github action is added in a workflow in this repo, we should be updating the spreadsheet and getting legal's approval if necessary)

Reply:
Oh I thought this was a different spreadsheet

Reply:
Do we need to put our own github actions into this sheet?

Reply:
yeah, I'm not 100% sure if we need to for our own actions. we could also put the package-lock for get-coverage-percent in Snyk, but not sure if that's necessary either. maybe @tmeyer2115 will have some thoughts on this

Reply:
this is true licenses are an incredibly crucial part of the software development lifecycle

Reply:
I will add Snyk after this pr is merged in